modules/cluster-and-ha-management/pages/ha-cluster.adoc

```diff
@@ -8,6 +8,11 @@
 
 TigerGraph HA service provides load balancing when all components are operational, as well as automatic fail over in the event of a service disruption.
 
+[TIP]
+====
+The high available cluster can tolerate the failure of **a single node**. To tolerate **multiple node** failures, please refer to xref:region-aware.adoc[Region Aware Cluster Configuration].
```

Comment:
This is not true, after region aware feature, we can tolerate more than one node down even for non-region aware case (if HA is on). Pls verify and confirm.

Reply:
I mean this is not accurate. I want to mention is, HA cluster can tolerate **ANY** one region down.